packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -103,10 +105,14 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
     [onEnterKeyDown, onKeyDown, supportNewline]
   );
 
+  let focusText;
```

Comment:
This can simply be `const focusText = (hasFocus === 'sendBoxTextField')`.

Even better, just inline to line 115? 

  `autoFocus={hasFocus === 'sendBoxTextField'}`

Reply:
yup this is better, will make it so!